client/my-sites/checkout/upsell-nudge/index.tsx

```diff
@@ -215,22 +215,24 @@ export class UpsellNudge extends Component< UpsellNudgeProps, UpsellNudgeState >
 	};
 
 	render() {
-		const { selectedSiteId, hasProductsList, hasSitePlans, upsellType } = this.props;
+		const { selectedSiteId, hasProductsList, hasSitePlans, upsellType, upgradeItem } = this.props;
 		const styleClass =
 			BUSINESS_PLAN_UPGRADE_UPSELL === upsellType
 				? 'business-plan-upgrade-upsell-new-design'
 				: upsellType;
 
+		// There is no `siteId` if we're purchasing a domain-only site, so we pass the site slug to the query component instead.
```

Comment:
Will that work since we call `parseInt()` on this value?

Reply:
Nice catch. Just fixed it.